infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/KubernetesTrustedCAProvisioner.java

```diff
@@ -33,32 +33,31 @@
 import org.eclipse.che.workspace.infrastructure.kubernetes.environment.KubernetesEnvironment;
 import org.eclipse.che.workspace.infrastructure.kubernetes.environment.KubernetesEnvironment.PodData;
 import org.eclipse.che.workspace.infrastructure.kubernetes.environment.KubernetesEnvironment.PodRole;
-import org.eclipse.che.workspace.infrastructure.openshift.project.OpenShiftProject;
+import org.eclipse.che.workspace.infrastructure.kubernetes.namespace.KubernetesNamespace;
 
 /**
- * Checks if config map with CA bundles is configured by specific property, and if it is, creates
- * map for ca bundles in workspace project, allowing Openshift to auto-inject values into it. (see
- * https://docs.openshift.com/container-platform/4.3/networking/configuring-a-custom-pki.html#certificate-injection-using-operators_configuring-a-custom-pki)
+ * Checks if config maps with CA bundles is configured by specific property. If they are, then
+ * creates single config map for all ca bundles in workspace namespace and mounts it into pods.
  */
 @Singleton
-public class Openshift4TrustedCAProvisioner {
+public class KubernetesTrustedCAProvisioner implements TrustedCAProvisioner {
 
-  public static final String CHE_TRUST_STORE_VOLUME = "che-self-signed-certs";
+  public static final String CHE_TRUST_STORE_VOLUME = "che-ca-certs";
```

Comment:
I am afraid of changing defaults

Reply:
This is only volume name for a workspace pod. I decided to change the name because old one does not reflect real purpose of it.